pages/dkp/kommander/2.2/install/air-gapped/index.md

```diff
@@ -175,14 +175,26 @@ export VERSION=v2.2.0
     wget "https://downloads.mesosphere.com/kommander/airgapped/${VERSION}/kommander_image_bundle_${VERSION}_linux_amd64.tar.gz" -O kommander-image-bundle.tar.gz
     ```
 
-1.  Place the bundle in a location where you can load and push the images to your private Docker registry.
+1.  Download the [DKP catalog applications][dkp_catalog_applications] image bundle if you intend on deploying any DKP catalog applications:
+
+    ```bash
+    wget "https://downloads.mesosphere.com/kommander/airgapped/${VERSION}/catalog_applications_image_bundle_${VERSION}_linux_amd64.tar.gz" -O catalog-applications-image-bundle.tar.gz
```

Comment:
These links don't work yet, but I assume that's just because they haven't been published yet?

Reply:
yep exactly!